app/scripts/lib/transaction/metrics.ts

```diff
@@ -1391,3 +1315,77 @@ function isInsufficientNativeBalance(
 
   return new Numeric(totalCost, 16).greaterThan(new Numeric(nativeBalance, 16));
 }
+
+function getMMSwapOrBridgeType(type: TransactionType): string | null {
+  if (type === TransactionType.swap) {
+    return 'mm_swap';
```

Comment:
Is this logic new? Can we define these in constants or a new enum such as `MetricsTransactionType`?